user-interface/src/main/java/life/qbic/datamanager/views/projects/project/access/AddCollaboratorToProjectDialog.java

```diff
@@ -167,6 +174,60 @@ public Registration addConfirmListener(ComponentEventListener<ConfirmEvent> list
     return addListener(ConfirmEvent.class, listener);
   }
 
+  /**
+   * A component displaying a user avatar and user name
+   */
+  public static class CollaboratorInfoRender extends Div {
+
+    private static final String ORCID_LOGO_PATH = "login/orcid_logo.svg";
+    private final Div userInfoContent;
+
+    public CollaboratorInfoRender(UserAvatar userAvatar,
+        String userName, String fullName) {
+      addClassName("collaborator-info");
+      userAvatar.setClassName("avatar");
+      userInfoContent = new Div();
+      userInfoContent.addClassName("user-info");
+      addInfoItem("Username", new Span(userName));
+      addInfoItem("Full Name", new Span(fullName));
+      add(userAvatar, userInfoContent);
+    }
+
+    private void setOrcid(String orcid) {
+      if (orcid.isEmpty()) {
+        return;
+      }
+      String orcidURL = "https://orcid.org/" + orcid;
```

Comment:
consider using the issuer url here

Reply:
It's completely overhauled to account for the possibility of multiple oidcs and reusability